.github/workflows/perl-openwsman.yml

```diff
@@ -0,0 +1,228 @@
+name: perl-openwsman
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - "dependencies/perl-openwsman/**"
+  push:
+    branches:
+      - develop
+      - master
+    paths:
+      - "dependencies/perl-openwsman/**"
+
+jobs:
+  get-environment:
+    uses: ./.github/workflows/get-environment.yml
+
+  package:
+    needs: [get-environment]
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - image: packaging-plugins-alma8
+            distrib: el8
+            package_extension: rpm
+            runner: ubuntu-22.04
+            arch: amd64
+          - image: packaging-plugins-alma9
+            distrib: el9
+            package_extension: rpm
+            runner: ubuntu-22.04
+            arch: amd64
+          - image: packaging-plugins-bullseye
+            distrib: bullseye
+            package_extension: deb
+            runner: ubuntu-22.04
+            arch: amd64
+          - image: packaging-plugins-bullseye-arm64
+            distrib: bullseye
+            package_extension: deb
+            runner: ["self-hosted", "collect-arm64"]
+            arch: arm64
+
+    runs-on: ${{ matrix.runner }}
+
+    container:
+      image: ${{ vars.DOCKER_INTERNAL_REGISTRY_URL }}/${{ matrix.image }}
+      credentials:
+        username: ${{ secrets.DOCKER_REGISTRY_ID }}
+        password: ${{ secrets.DOCKER_REGISTRY_PASSWD }}
+
+    name: package ${{ matrix.distrib }} ${{ matrix.arch }}
+
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+
+      - name: Install locally sblim-sfcc
+        run: |
+          if [ "${{ matrix.distrib }}" = "bullseye" ]; then
+            apt-get update
+            apt-get install -y libcurl4-openssl-dev
+          else
+            dnf install -y wget automake libtool libcurl-devel gcc-c++
+          fi
+
+          cd /tmp
+          wget -O - https://github.com/kkaempf/sblim-sfcc/archive/refs/tags/SFCC_2_2_8.tar.gz|tar zxvf -
+          cd sblim-sfcc-SFCC_2_2_8
+          ./autoconfiscate.sh
+          ./configure --prefix=/usr
+          make
+          make install
+        shell: bash
+
+      - name: Build openwsman
+        run: |
+          if [ "${{ matrix.distrib }}" = "bullseye" ]; then
+            apt-get install -y cmake libssl-dev libpam-dev swig libxml2-dev
+          else
+            if [ "${{ matrix.distrib }}" = "el9" ]; then
+              dnf install -y 'dnf-command(config-manager)'
+              dnf config-manager --set-enabled crb
+            fi
+            dnf install -y cmake pam-devel swig libxml2-devel openssl-devel
+          fi
+
+          cd /tmp
+          wget -O - https://github.com/Openwsman/openwsman/archive/refs/tags/v2.7.2.tar.gz|tar zxvf -
+          cd openwsman-2.7.2/
```

Comment:
Once all the perl dependencies workflows will be moved to this repo, we'll need to extract the version from packaging yaml file.
This will avoid to duplicate version number between workflows and packaging files